frontend/src/components/common/comment-item/comment-item.tsx

```diff
@@ -9,16 +10,23 @@ import { AppRoute } from 'common/enums/enums';
 type Props = {
   hasTimestamp?: boolean;
   comment: Comment;
+  user?: User | null;
```

Comment:
instead of the full user, we need to pass a boolean flag
```suggestion
 isOwner: boolean;
```